.pre-commit-config.yaml

```diff
@@ -4,6 +4,13 @@ repos:
   hooks:
   - id: black
 
+- repo: https://github.com/keewis/blackdoc
+   rev: v0.3.8
+   hooks:
+   - id: blackdoc
+     files: '\.py$'
+     exclude: pyvista/plotting/charts.py
```

Comment:
What's wrong with charts?

Reply:
The use of `docsubs`:

https://github.com/pyvista/pyvista/blob/a26a52d037a265774be0ca06bb141e3346e73ad0/pyvista/plotting/charts.py#L1773-L1781

I can’t think of a workaround for this. We may consider opening an issue on https://github.com/keewis/blackdoc and seeing if we can implement a skip.

Reply:
Issue opened:
https://github.com/keewis/blackdoc/issues/172